js/aes_gcm.js

```diff
@@ -0,0 +1,104 @@
+/* Copyright 2019 The Subscribe with Google Authors. All Rights Reserved.
```

Comment:
This should be:

```
/**
 * Copyright
```

I know it's annoying, but I believe this matters for some transforming tools.

Reply:
Done.